nuitka/freezer/DllDependenciesMacOS.py

```diff
@@ -201,10 +203,23 @@ def _resolveBinaryPathDLLsMacOS(
             if candidate is not None and os.path.exists(candidate):
                 resolved_path = candidate
 
+        def getRightActivePlugin(required_plugin_name):
+            for active_plugin in getActivePlugins():
+                if required_plugin_name == active_plugin.plugin_name and hasattr(
+                    active_plugin, "getListSkippedDLLs"
```

Comment:
So what I would do is to have an implementation that returns an empty tuple in the base class of all plugins, then no `hasattr` check is needed. And I would pass `package_name` to the plugin, so it can decide for itself, if it's responsible or not.

Reply:
Added a base `getListSkippedDLLs`method, removed`hasattr`check.

Reply:
Yes, that's always a good idea. DLLs must not be globally banned or anything, just for the module(s) that a plugin is responsible for. Eventually I want to make it so that modules register for namespaces, and not act globally. Not having that argument will then be an issue.